src/EnergyPlus/DaylightingManager.cc

```diff
@@ -5647,18 +5608,16 @@ inline int findWinShadingStatus(EnergyPlusData &state, int const IWin)
 }
 
 void DayltgGlare(EnergyPlusData &state,
-                 int &IL,        // Reference point index: 1=first ref pt, 2=second ref pt
-                 Real64 &BLUM,   // Window background (surround) luminance (cd/m2)
-                 Real64 &GLINDX, // Glare index
-                 int &ZoneNum    // Zone number
+                 int &IL,                  // Reference point index: 1=first ref pt, 2=second ref pt
```

Comment:
Are all of these output parameters? The ones that are input parameters should be passed by value. If there is an output parameter (`GLINDX`?) it should be returned via the return value.

Reply:
> Are all of these output parameters? 

Honestly, I don't know for sure. This pattern is present in many of the daylighting functions, and it does appear that some of these indexes get changed along the way and the next function gets the new value as an input. As this exercise has revealed, there is some/much/immense room for improvement in the daylighting code.